blog/_posts/2021-07-06-distributed-rdataframe.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "RDataFrame is going distributed!"
+layout: archive
+author: Vincenzo Eduardo Padulano
+---
+
+A recently introduced Python package enables distributing ROOT RDataFrame workloads to a set of remote resources.
+This feature is available in experimental phase since the latest ROOT 6.24 release, allowing users to write and run
+their applications from within the same interface while steering the computations to an Apache Spark cluster.
+
+## In detail
+
+[RDataFrame](https://root.cern/doc/master/classROOT_1_1RDataFrame.html) is ROOT's high-level interface for data
+analysis and since its release in 6.14 has seen tons of usages in real world analysis use cases and other examples in
+the wild. Parallelism has always been a staple of its design with support for executing the event loop on all cores
+of a machine thanks to implicit multi-threading. Since ROOT 6.24, this aspect of RDataFrame has been enhanced further
+with distributed computing capabilities, allowing users to run their analysis on multi-node computing clusters through
+widely used frameworks such as [Apache Spark](https://spark.apache.org/), [Dask](https://dask.org/) or
+[AWS Lambda](https://aws.amazon.com/lambda/). Here is an example usage of the tool (requires the Python `pyspark`
+package):
+
+```python
+import ROOT
+ 
+# Point RDataFrame calls to the Spark specific RDataFrame
+RDataFrame = ROOT.RDF.Experimental.Distributed.Spark.RDataFrame
+ 
+# It still accepts the same constructor arguments as traditional RDataFrame
+df = RDataFrame("mytree", "myfile.root")
+ 
+# Continue the application with the traditional RDataFrame API
+sum = df.Filter("x > 10").Sum("y")
+h = df.Histo1D("x")
+ 
+print(sum.GetValue())
+h.Draw()
+```
+
+The main goal is to support distributed execution of any RDataFrame application. This has led to the creation of a 
+**Python** package that connects the RDataFrame API (available in Python through PyROOT) and the APIs of
+distributed computing frameworks, which are offered in Python in the vast majority of cases. Another key goal is to be
+able to support as many backends as possible, in order to avoid being tied to any particular infrastructure. This is
```

Comment:
Users are tied because they will have one infra - that "tying" is a developer's pov. What about:
```suggestion
able to support as many backends as possible, in order to provide a solution for as many users as possible. This is
```

Reply:
This is slightly clashing with Enric's comment from above. As I wrote there, I can maybe "merge" the two comments and write:
`Another key goal is to offer a variety of backends, to provide a solution to a variety of use cases` ?